### PR TITLE
remove late penalties and fix issue with withdraw divs

### PR DESCRIPTION
**Pull request by Dellybro**

### CODE REVIEW

File: deployed-addresses/addresses.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "AUCTION_ADDRESS": "0x5a756C7e7b1ed926034e09cc91259684145b1313",
   "TOKEN_ADDRESS": "0x51F8dC388D196c80E2f4edFBc1391E34e7a65928",
   "NATIVESWAP_ADDRESS": "0xCA2EB609DEE593b4e093C98d6006C23d96122018",
-  "BPD_ADDRESS": "0xCA2EB609DEE593b4e093C98d6006C23d96122018",
+  "BPD_ADDRESS": "0x9905D0FCF3149C08386204e6d8eB2803E4ff54e8",
```
Comment (sigurdpl) [inline deployed-addresses/addresses.json:6]: ropsten ?

File: deployed-addresses/addresses.json [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
   "FOREIGNSWAP_ADDRESS": "0x74e33e18620e6c7C0aD2C58Ca650679334642f7C",
   "SUBBALANCES_ADDRESS": "0x5b9f4674B252AEde1Ac581aa0C9d6655240f4C37",
   "STAKING_ADDRESS": "0xc2272F8D6D10a165325c3b230BA6641d624ba67f",
-  "AUCTION_MANAGER_ADDRESS": "0x693907978Dcd3Ef38A2cE68c353bf1a13203Fc86",
+  "AUCTION_MANAGER_ADDRESS": "0x7f9db4e2a101b1ee7b5af643d32e2f80f38e95ff",
```
Comment (sigurdpl) [inline deployed-addresses/addresses.json:10]: ropsten ?

**Review by sigurdpl — COMMENTED** [review]
PR looks good, there are two changes in contract deployment addresses

**Review by petec86 — APPROVED** [review]

File: deployed-addresses/addresses.json (hunk 1 above)
Comment (Mvii7) [reply deployed-addresses/addresses.json:6]: doesnt matter

File: deployed-addresses/addresses.json (hunk 2 above)
Comment (Mvii7) [reply deployed-addresses/addresses.json:10]: doesnt matter

**Pull request merged**